lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -44,6 +44,11 @@
      description:
      - UUID of the instance to manage if known, this is VMware's unique identifier.
      - This is required if name is not supplied.
```

Comment:
if name or moid is not...
and change line 37

Reply:
Thanks for the catch, updated!